util.go

```diff
@@ -91,13 +91,23 @@ func insensitiviseMap(m map[string]interface{}) {
 func absPathify(inPath string) string {
 	jww.INFO.Println("Trying to resolve absolute path to", inPath)
 
-	if strings.HasPrefix(inPath, "$HOME") {
+	if strings.HasPrefix(inPath, "$HOME") &&
```

Comment:
Could this be rewritten as something like `inPath == "$HOME" || strings.HasPrefix(inPath, "$HOME"+os.PathSeparator)` or equivalent (or even using some sort of splitting)? The current condition is not really expressive.

Side note: looking at the `path/filepath` package there isn't much that can help here, but ideally it would be nice.